parse/index.js

```diff
@@ -381,10 +387,26 @@ export default ({
             currentData.validate = getDataValidate(p)
           } else if (p.name === 'required') {
             currentData.validate.required = p.value
+          } else if (p.name === 'aggregate') {
+            currentAggregateData.aggregate = getDataAggregate(p)
+          } else if (p.name !== 'data') {
+            if (
+              !currentAggregateData.aggregate &&
+              currentAggregateData.data.length > 1
+            ) {
+              warnings.push({
+                type: `No aggregate function was provided, but ${currentAggregateData.data.length} data keys were found. Did you forget to specify an aggregate function?`,
+                line,
+                loc: block.loc,
+              })
+            }
+            currentAggregateData = null
```

Comment:
Not sure how feasible (and useful) it is given the code structure but before resetting this to null, do we want/need to set the end `loc` of `currentAggregateData` to match the last bit of data?

Reply:
yes, I can do that. didn't add it as for now only the start is used